src/applications/static-pages/scroll-top-button.js

```diff
@@ -6,6 +6,9 @@
  */
 
 function navigateToTop() {
+  if (document.getElementById('content')) {
```

Comment:
Can we cache this variable?

```
const content = document.getElementById('content');
if (content) {
    content.focus();
  }
```

Reply:
I think this element is also non-focusable, so we'll need to add `tabindex="-1"` as well so set focus

Reply:
done, thanks for the feedback!